linux/debian/control

```diff
@@ -27,7 +27,9 @@ Build-Depends: debhelper (>= 11.2),
                qmake6 (>=6.2.0~),
                qt6-base-dev (>=6.2.0~),
                qt6-base-dev-tools (>=6.2.0~),
+               qt6-base-private-dev (>=6.2.0~),
```

Comment:
For my info, could you please explain why these packages with private headers are needed?

Reply:
https://github.com/mozilla-mobile/mozilla-vpn-client/pull/7203

We currently have an influx of crashes that are triggered from "something" called in the js-engine. 
So in sentryadapter we added a crashandler that can hook into the JS stack and produce a mini stacktrace - that will be attached in a sentry report. 
Those api's are not public. Since this pr will enable us to also ship sentry on linux we know have a dependency on both private qt and qml headers :)